src/js/dbSNP-import/dbSNP-import-progress.js

```diff
@@ -74,13 +74,30 @@ EvadbSNPImportProgress.prototype = {
                 '<p>This report allows you to track the progress of the dbSNP data import.' +
                 '<p>' +
                     'Variants will be available in the Variant Browser if they satisfy the <a href="?Submit-Data">EVA submission requirements</a>. ' +
-                    'dbSNP variants that don\'t satisfy these requirements will still be imported, and searchable via a separate web view and API. ' +
+                    'dbSNP variants that don\'t satisfy these requirements will still be imported, and searchable via a separate ' +
+                    '<a href="?Home">web view</a> and <a href="https://www.ebi.ac.uk/eva/webservices/identifiers/swagger-ui.html">API</a>, ' +
+                    'and downloadable from our <a href="ftp://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/">FTP</a>. ' +
+                    'Variants that don\'t map to any INSDC assembly are present only in our FTP at the moment. ' +
                     'We will work to make this experience as intuitive as possible, while keeping our commitment to only make high-quality variants part of the core EVA database.' +
                 '</p>'+
                 '<p> ' + 
                     'In addition to the most recent RS IDs available for a given species, ' + 
                     'older RS IDs that were merged into the newer ones will also be imported to support reproducible analyses based on historical data.' +
                 '</p>' +
+                '<p> ' +
+                    '<h3>Notes</h3>' +
+                    'Human RS IDs imported: 666772339 out of 666783996 (shown as 99.99%):' +
+                    '<ul>' +
+                        '<li>7941 could not be imported because they did not have mappings on grch38</li>' +
+                        '<li>3716 could not be imported because they did not have top-level placement confirmation ' +
+                        '(i.e., locus could not be definitively identified).</li>' +
+                    '</ul>' +
+                    'No Human SS IDs were imported due to absence of batch handles in the JSON data release ' +
```

Comment:
Indicate that it is a dbSNP release. Is there a link or anything we can add?

Reply:
modified the text. I have no idea of any useful link here. @sundarvenkata-EBI ?

Reply:
@andresfsilva good point! Please use: ftp://ftp.ncbi.nih.gov/snp/latest_release/JSON as the link.